Yoast/wordpress-seo
# Create an alternative to the app.refresh

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Adds `wp-seo-analysis-worker.js` to its own Webpack path so that we can create the worker from that URL. The url is passed from PHP to JS in `wpseoAnalysisWorkerL10n`.
* Exposes the AnalysisWorkerWrapper to `window.YoastSEO.analysisWorker`.

## Questions

* What should happen if an error occurred in the worker?

## Test instructions

This PR can be tested by following these steps:

* Build & run it.
* Look if the worker configuration matches the settings (change it in WP and see if it changed too):
  - translations
  - locale
  - contentAnalysisActive
  - keywordAnalysisActive

## Quality assurance


Fixes https://github.com/Yoast/YoastSEO.js/issues/1677
